### PR TITLE
changelog

### PR DESCRIPTION
**Pull request by jburel**
Introduce a changelog
"Migrate" releases section from figure.openmicroscopy to a changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+## 2.0.1 (2016-10-26)
+
+Features:
+
+	- Distribute app from Pypi
```
Comment (mtbc) [inline CHANGELOG.md]: capitalization is "PyPI"

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+## 2.0.0 (2016-10-11)
+
+Features:
+
+	- Rename repository
+	- Re-organize packages
```
Comment (mtbc) [inline CHANGELOG.md]: "Reorganize" (no hyphen)

**Comment by jburel** [comment]
@will-moore: any comments? 
If not we can merge so I can move forward with the update of figure.o.o

**Comment by will-moore** [comment]
Sorry, Yes - looks good. Thanks.

**Comment by jburel** [comment]
Thanks Merging.

**Pull request merged**